app/src/main/res/layout/fragment_add_on_details.xml

```diff
@@ -21,115 +21,143 @@
             android:id="@+id/details"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="20dp"
             android:textColor="?primaryText"
             android:textColorLink="?aboutLink"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
             tools:text="@tools:sample/lorem/random" />
 
         <TextView
             android:id="@+id/author_label"
+            style="@style/AboutHeaderContentText"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_below="@+id/details"
-            style="@style/AboutHeaderContentText"
-            android:text="@string/mozac_feature_addons_authors" />
+            android:text="@string/mozac_feature_addons_authors"
+            android:layout_marginTop="20dp"
+            app:layout_constraintEnd_toStartOf="@+id/author_text"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintHorizontal_chainStyle="spread_inside"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/details" />
 
         <TextView
             android:id="@+id/author_text"
+            style="@style/AboutHeaderContentText"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_below="@+id/details"
             android:layout_alignParentEnd="true"
-            style="@style/AboutHeaderContentText"
+            android:layout_marginTop="20dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toEndOf="@+id/author_label"
+            app:layout_constraintTop_toBottomOf="@+id/details"
             tools:text="@tools:sample/full_names" />
 
         <View
             android:id="@+id/author_divider"
             android:layout_width="match_parent"
             android:layout_height="1dp"
-            android:layout_below="@+id/author_text"
             android:layout_marginTop="10dp"
-            android:layout_marginBottom="10dp"
             android:background="?android:attr/listDivider"
-            android:importantForAccessibility="no" />
+            android:importantForAccessibility="no"
+            app:layout_constraintTop_toBottomOf="@id/author_label"
```

Comment:
Nit: Dividers should be constrained to the view that could expand on multiple rows (or a barrier), so in our case here, `author_text`